.github/workflows/android_tests.yml

```diff
@@ -110,5 +110,5 @@ jobs:
             --test_output=all \
             --build_tests_only \
             $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-linux-clang") \
-            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+             --copt=-DALLOW_TCP_LISTENERS --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
```

Comment:
Can this be moved to a `.bazelrc` config?

Reply:
The goal here is to have E-M build without TCP listeners, but we exempt it for this CI due to the proxying tests.  will comment inline.

Reply:
But then these tests will fail when run locally?

Why not add `build test --copt=-DALLOW_TCP_LISTENERS` to the `.bazelrc` so it works both on CI and locally?

Reply:
because then all tests will always run with TCP listeners, and we won't have regression testing for, say, the C++ tests running without TCP listeners.

What I'm hearing is you'd prefer I add it to the bazelrc, and explicitly not allow TCP listeners on all the non-legacy CI instead?

Reply:
Hmm, yeah I was thinking a config that we could pass when running these tests specifically, but then it doesn't make much difference if you have to pass a config or a `--copt` because either way `./bazelw test ...`  won't be sufficient.

I'm ok keeping this as-is then.